src/ray/common/task/scheduling_resources.h

```diff
@@ -21,6 +21,8 @@ const std::string kGPU_ResourceLabel = "GPU";
 const std::string kObjectStoreMemory_ResourceLabel = "object_store_memory";
 const std::string kMemory_ResourceLabel = "memory";
 
```

Comment:
nit, but can you remove the newline here? (put all these variables together without an empty line)